File: caitlin-schaeffer/day-04/remove-string-spaces.js
/*
Params: Write a function that removes the spaces from the string, then return the resultant string.
Return: a string with no spaces
Solution: Using the replace() method, by passing a regular expression matching any space as the first argument, and an empty string ('') as the second.
The RegExp object is used for matching text with a pattern. So we are aligning the string with the regex (g) and running it twice, where the empty string removes the spacing.
*/

function noSpace(x) {
    let noSpace = x.replace(/ /g, '');
    return noSpace
}

/* Ember's Feedback:
  Please clean these up before you submit.
  Good use of a Regular Expression, they can be very powerful!
  Let me know if you have questions about RegEx.
*/

File: ryan-anderson/day-31/remove-string-spaces.py
def no_space(x):
    return x.replace(" ", "")


# use .replace to remove spaces

"""
    Ember's Feedback:
    - good work!
"""


# Alternative Solution
def no_space(x):
    return "".join(x.split())


def no_space(x):
    import re

    return re.sub("\s*", "", x)
